### PR TITLE
Delete generated services & service tests before running CI

### PR DESCRIPTION
**Pull request by jbelkins**
## Description of changes
Prevents the entire SDK from being built and tested during CI runs by deleting generated services and their test suites before starting CI.  Then, when the SDK is built & tested, only runtime components are built and tested.

While testing the full, built AWS SDK is generally desirable, it takes too much time as-is to complete during routine CI runs.  #808 tracks the need to establish a CI task to monitor the health of the full SDK.

## New/existing dependencies impact assessment, if applicable
No new dependencies were added to this change.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: builder.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
     "gradlew": "{source_dir}/gradlew"
   },
   "pre_build_steps": [
-    "cd {source_dir}",
```
Comment (ganeshnj) [inline builder.json:9]: could you add log (plain echo) here, why are we doing this for posterity.

**Review by ganeshnj — APPROVED** [review]

**Review by epau — APPROVED** [review]
assuming ganesh's comment is addressed, lgtm

File: builder.json (hunk 1 above)
Comment (ganeshnj) [reply builder.json:9]: and put a link to https://github.com/awslabs/aws-sdk-swift/issues/808

Comment (jbelkins) [reply builder.json:9]: Done

**Pull request merged**